modules/rosa-creating-cluster.adoc

```diff
@@ -17,11 +17,13 @@ You have completed the installation prerequisites.
 
 Enter the following command to create your cluster with the default cluster settings.
 
+* To view other options when creating a cluster, enter `rosa create cluster --help`.
```

Comment:
The above sentence of "Enter the following command" is now pretty far removed from the actual command of `$ rosa create cluster --cluster-name=rh-rosa-test-cluster1`

I don't know if there was a reason for removing it, but I myself would probably put the tips from the bullets after the command. Because your sentence says "with the default cluster settings". And then the bullets are just giving tips on how to further customize outside of the default settings.